docs/sources/project/work-issue.md

```diff
@@ -147,57 +147,7 @@ You should pull and rebase frequently as you work.
 
 1. Return to the terminal on your local machine.
 
-2. Make sure you are in your branch.
-
-		$ git checkout 11038-fix-rhel-link
-
-3. Fetch all the changes from the `upstream master` branch.
-
-		 $ git fetch upstream master
-
-  	This command says get all the changes from the `master` branch belonging to
-  	the `upstream` remote.
-
-4. Rebase your master with the local copy of Docker's `master` branch.
-
-		 $ git rebase -i upstream/master
-   
-  	This command starts an interactive rebase to rewrite all the commits from
-	Docker's `upstream/master` onto your local branch, and then re-apply each of
-	your commits on top of the upstream changes. If you aren't familiar or
-  	comfortable with rebase, you can <a
-  	href="http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-
-  	rebase" target="_blank">learn more about rebasing</a> on the web.
-  
-5. Rebase opens an editor with a list of commits.
-
-			pick 1a79f55 Tweak some of the other text for grammar 
-			pick 53e4983 Fix a link 
-			pick 3ce07bb Add a new line about RHEL
-        
-  	If you run into trouble, `git --rebase abort` removes any changes and gets
-  	you back to where you started. 
-
-6. Squash the `pick` keyword with `squash` on all but the first commit.
-
-			pick 1a79f55 Tweak some of the other text for grammar
-			squash 53e4983 Fix a link
-			squash 3ce07bb Add a new line about RHEL
-
-  	After closing the file, `git` opens your editor again to edit the commit
-  	message. 
-
-7. Edit the commit message to reflect the entire change.
-
-	Make sure you include your signature.
-
-8. Push any changes to your fork on GitHub.
-
-	The rebase rewrote history, so you'll need to use the `-f` or `--force` flag
-	to push your change.
-
-		$ git push -f origin 11038-fix-rhel-link
-
+2. Follow the [rebase instructions](/project/create-pr/#rebase-your-branch).
```

Comment:
Ok, so here is where I would prefer not to do this.  If this tutorial were intended purely as a reference  repetition would be something I'm likely to avoid. However, I design these pieces more along the lines of interactive, online training not a user guide writing piece.

When designing a training piece, repeating information at different points in the flow is a good thing. It allows for more in-depth learning -- users are learning not only the act of rebasing but also that rebasing needs to be done several times -- unlike, say, creating a branch.  

Finally, in a tutorial, I only link out of a step when forced to --- say if the first step is to install and install is complex.  This keeps the person I am training in the flow of the process rather than risking them losing their place by stepping out of the process.